src/Http/Routing/src/Builder/MapActionEndpointRouteBuilderExtensions.cs

```diff
@@ -76,5 +83,203 @@ public static class MapActionEndpointRouteBuilderExtensions
 
             return new MapActionEndpointConventionBuilder(conventionBuilders);
         }
+
+        /// <summary>
+        /// Adds a <see cref="RouteEndpoint"/> to the <see cref="IEndpointRouteBuilder"/> that matches HTTP GET requests
+        /// for the specified pattern.
+        /// </summary>
+        /// <param name="endpoints">The <see cref="IEndpointRouteBuilder"/> to add the route to.</param>
+        /// <param name="pattern">The route pattern.</param>
+        /// <param name="action">The delegate executed when the endpoint is matched.</param>
+        /// <returns>A <see cref="IEndpointConventionBuilder"/> that can be used to further customize the endpoint.</returns>
+        public static MapActionEndpointConventionBuilder MapGet(
```

Comment:
Aren't there already Map* extension methods on route builder? Do these not colide because of `Delegate` parameter?

Reply:
Nope.

Reply:
These are overloads, but they should only be used for delegates that aren't `RequestDelegate`s. This was also mentioned in https://github.com/dotnet/csharplang/pull/4451#issuecomment-785550046.